test/unit/network_spec.js

```diff
@@ -69,11 +69,10 @@ describe('network', function() {
     // The test is valid for FF only: the XHR has support of the
```

Comment:
These comments can now be removed, including the TODO you fixed with this patch.